storage/engine/engine.go

```diff
@@ -100,12 +100,12 @@ type Engine interface {
 	Merge(key proto.EncodedKey, value []byte) error
 	// Capacity returns capacity details for the engine's available storage.
 	Capacity() (proto.StoreCapacity, error)
-	// SetGCTimeouts sets a function which yields timeout values for GC
-	// compaction of transaction and response cache entries. The return
-	// values are in unix nanoseconds for the minimum transaction row
-	// timestamp and the minimum response cache row timestamp respectively.
-	// Rows with timestamps less than the associated value will be GC'd
-	// during compaction.
+	// SetGCTimeouts sets timeout values for GC compaction of
+	// transaction and response cache entries. The values are
+	// specified in unix nanoseconds for the minimum transaction
```

Comment:
"unix nanoseconds" is awkward. can you clarify that the intention is "unix time in nanoseconds"?